target/linux/ar71xx/image/generic.mk

```diff
@@ -229,6 +229,78 @@ define Device/dragino2
 endef
 TARGET_DEVICES += dragino2
 
+define Device/e1700ac-8M
+  DEVICE_TITLE := WHQX E1700AC-8M (8MB flash)
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-ledtrig-usbport kmod-ath10k ath10k-firmware-qca988x
+  BOARDNAME := E1700AC
+  IMAGE_SIZE := 7744k
+  KERNEL_SIZE := 1472k
+  ROOTFS_SIZE := 6336k
+  MTDPARTS := spi0.0:256k(u-boot)ro,64k(u-boot-env)ro,6336k(rootfs),1472k(kernel),64k(art)ro,7744k@0x50000(firmware)
+  IMAGE/sysupgrade.bin := append-rootfs | pad-rootfs | pad-to $$$$(ROOTFS_SIZE) | append-kernel | check-size $$$$(IMAGE_SIZE)
```

Comment:
You move all your devices in upgrade/platform.sh touse the metadata, but You not add "append-metadata" to sysupgrade image.
Split this line so that it does not exceed 80 characters. Look to line 86-87 in this file